SmartPlaces.Facilities/samples/Topology/README.md

```diff
@@ -49,3 +49,86 @@ The following secrets must be defined in the KeyVault in order to run this code:
 ## Logging and Metrics
 
 Logs and Metrics will appear in the Application Insights instance pointed to by the App Insights connection string. For more information on how to configure the Application Insights SDK, please see: https://learn.microsoft.com/en-us/azure/azure-monitor/app/worker-service
+
+## Monitoring and Debugging the Topology Process
+
+There are three common ways to monitor / debug the topology loading process:
+
+1. If you are running the app inside an Azure container, go to the container in the Azure portal, and click on the logs tab in the Containers portal. This will give you a look at the last few minutes of the log. 
```

Comment:
While this is the most beneficial to us and the quickest for troubleshooting, we've not given deployment guidance. A customer could just as easily deploy the code in a webapp or their own kubernetes cluster. We should probably try to order the readme advice on what we've provided as sample code first as opposed to guidance on assumptions they arrive at the same architectural decisions that we made.